docs/docs/general/06-Tutorials/custom-element.md

```diff
@@ -0,0 +1,260 @@
+# Custom Elements in askui
+
+
+
+## Overview
+
+**Custom Element Selection** is a feature in askui that enables you to create custom selectors for elements on the screen, instead of relying on the standard selectors provided such as **Button**, **Textfield**, etc.
+
+With this feature, you can define a custom selector based on how the element is displayed on the screen. This can be particularly useful in situations where standard selectors are unreliable due to the non-standard properties of the element. It provides greater flexibility and control, allowing you to tailor the automation to meet the specific needs of your application.
+
```

Comment:
Please add an admonition (https://docusaurus.io/docs/markdown-features/admonitions) here that states this technique should not be used except absolutely necessary, because it has a negative performance impact.